Makefile

```diff
@@ -54,7 +54,7 @@ endif
 
 
 ifeq (${IMAGE_TAG}, )
-  IMAGE_TAG = ci
+  IMAGE_TAG = 2.7.0-develop
```

Comment:
Should we have a file on the root where we define the version? Or perhaps simply read the version from the chart?

Reply:
We can read it from the chart that should be doable. But would need yq for that.

Reply:
That means the tests scripts would now run in nix-shell?

Reply:
You could still run them outside nix-shell, as long as yq is present?

Reply:
Buy yq is coming from nix-shell if you see. That's the problem. Can we do something about that?